web/src/features/moreCast2/components/MoreCast2DataGrid.tsx

```diff
@@ -33,47 +33,60 @@ const MoreCast2DataGrid = (props: MoreCast2DataGridProps) => {
   const { rows } = props
   const loading = useSelector(selectMorecast2TableLoading)
 
-  const gridColumnDefGenerator = (field: string, headerName: string, precision: number) => {
-    return {
-      field: field,
-      disableColumnMenu: true,
-      disableReorder: true,
-      editable: true,
-      headerName: headerName,
-      sortable: false,
-      type: 'number',
-      width: 120,
-      valueFormatter: (params: GridValueFormatterParams) => predictionItemValueFormatter(params, precision),
-      valueGetter: (params: GridValueGetterParams) => predictionItemValueGetter(params),
-      valueSetter: (params: GridValueSetterParams) => predictionItemValueSetter(params, field)
+  const numFormatter = (value: number, precision: number) => {
+    if (isUndefined(value) || isNaN(value)) {
+      return NaN
     }
+
+    const multiplier = Math.pow(10, precision || 0)
+    return Math.round(value * multiplier) / multiplier
```

Comment:
Can we just use `value.toFixed(precision)`?

Reply:
I'm with Brady - .toFixed() is the way to go

Reply:
We might be able to do that, toFixed() returns a string though and the component might need a number. I'll check it out.

Reply:
you should just be able to do `Math.round(value, multiplier)`

Reply:
https://github.com/bcgov/wps/pull/2699/commits/1c5f77cbf4b89abf19ddbcb2815122672a2ccaed